source/installation_guide/upgrading/from-2.3-to-3.0.rst

```diff
@@ -44,4 +44,31 @@ Removed features and their replacements
+| | fts-squat                   |                                                                                          |
++-------------------------------+------------------------------------------------------------------------------------------+
+| Weak password schemes         | Weak password schemes are disabled by default, you need to use                           |
+|                               | :dovecot_core::ref:`auth_allow_weak_schemes` to enable them.                             |
++-------------------------------+------------------------------------------------------------------------------------------+
+| Global ACL directory          | Use global acl file instead.                                                             |
++-------------------------------+------------------------------------------------------------------------------------------+
+| ``ssl-parameters.dat``        | This file is no longer converted automatically by config process, you need to set        |
+|                               | :dovecot_core:ref:`ssl_dh` setting if you need non-ECC Diffie-Hellman.                   |
++-------------------------------+------------------------------------------------------------------------------------------+
+| License plugin                | This plugin has been removed and ``license_checksum`` setting is marked obsolete         |
++-------------------------------+------------------------------------------------------------------------------------------+
+| shadow auth driver            | Use :ref:`authentication-passwd` instead.                                                |
++-------------------------------+------------------------------------------------------------------------------------------+
+| old-stats plugin              | Use new stats instead.                                                                   |
++-------------------------------+------------------------------------------------------------------------------------------+
+| Memcached dict driver         | Use redis instead.                                                                       |
++-------------------------------+------------------------------------------------------------------------------------------+
+| IMAP: Remove X-CANCEL command | Not used by anything.                                                                    |
++-------------------------------+------------------------------------------------------------------------------------------+
+| IMAP: Remove X-STATE command  | Not used by anything.                                                                    ¦
++-------------------------------+------------------------------------------------------------------------------------------+
+| dsync: Remove -D parameter    | It hasn't been necessary for a long time, and it is broken                               |
```

Comment:
This sounds like the doveadm -D debug parameter. Should clarify somewhere that it's the parameter that "disables syncing mailbox renames".